aspnetcore/release-notes/aspnetcore-6.0.md

```diff
@@ -0,0 +1,704 @@
+---
+title: What's new in ASP.NET Core 6.0
+author: rick-anderson
+description: Learn about the new features in ASP.NET Core 6.0.
+ms.author: riande
+ms.custom: mvc
+ms.date: 10/29/2021
+no-loc: [Home, Privacy, "ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR, Kestrel]
+uid: aspnetcore-6.0
+---
+# What's new in ASP.NET Core 6.0
+
+This article highlights the most significant changes in ASP.NET Core 6.0 with links to relevant documentation.
+
+## ASP.NET Core MVC and Razor improvements
+
+## Minimal APIs
+
+Minimal APIs are architected to create HTTP APIs with minimal dependencies. They are ideal for microservices and apps that want to include only the minimum files, features, and dependencies in ASP.NET Core. For more information, see <xref:tutorials/min-web-api>.
+
+## SignalR
+
+### SignalR message size
+
+Size limitations on SignalR message size for Blazor Server apps have been removed.
+
+### Long running activity tag for SignalR connections
+
+SignalR uses the new <xref:Microsoft.AspNetCore.Http.Features.IHttpActivityFeature.Activity?displayProperty=fullName> to add an `http.long_running` tag to the request activity. `displayProperty` is used by [APM services](https://wikipedia.org/wiki/Application_performance_management) like [Azure Monitor Application Insights](/azure/azure-monitor/azure-monitor-app-hub) to filter SignalR requests from creating long running request alerts.
```

Comment:
What is `displayProperty`? I don't see it on [Activity](https://github.com/dotnet/runtime/blob/main/src/libraries/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/Activity.cs).

Reply:
Typo, fixed.